dashboards/kubernetes_capacity_planning/README.md

```diff
@@ -0,0 +1,25 @@
+# Kubernetes Capacity Planning
+
+The goal of this dashboard is to identify clusters with the biggest waste of resources related to kubernetes miscronfiguration. The second objective is to then highlight the biggest offenders to drive change.
```

Comment:
The goal of this dashboard is to identify clusters with the biggest waste of resources related to kubernetes misconfiguration. The second objective is to then highlight the biggest offenders to drive change.

Reply:
typo in misconfiguration